src/lxc/tools/utils.c

```diff
@@ -0,0 +1,9 @@
+
+#include "tools/utils.h"
```

Comment:
Both files, `utils.{c,h}` need to gain a license a top anything else as well. You can add your name and email there but please also include the others of the original function in the license header. An example would be:

```
/*
 * Copyright © 2017 Ricardo Sanchez A <ricardo.sanchez@utexas.edu>.
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License version 2, as
 * published by the Free Software Foundation.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License along
 * with this program; if not, write to the Free Software Foundation, Inc.,
 * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
```

However, I'm not an attorney. :)